packit_service/events/koji/result.py

```diff
@@ -219,13 +222,17 @@ def identifier(self) -> str:
 
     @classmethod
     def from_event_dict(cls, event: dict) -> "Task":
+        request = nested_get(event, "info", "request", default=[])
+        target_branch = request[1] if len(request) >= 2 else None
+
         return Task(
             task_id=event.get("task_id"),
             state=KojiTaskState(event.get("state")) if event.get("state") else None,
             old_state=(KojiTaskState(event.get("old_state")) if event.get("old_state") else None),
             rpm_build_task_ids=event.get("rpm_build_task_ids"),
             start_time=event.get("start_time"),
             completion_time=event.get("completion_time"),
+            target_branch=target_branch,
```

Comment:
I don't think the `event` dict contains `request`, but it should contain `target_branch`:

```suggestion
            target_branch=event.get("target_branch"),
```